src/ArtisanApartmentCommands.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace BuildEmpire\Apartment;
+
+use BuildEmpire\Apartment\Exceptions\SchemaAlreadyExistsException;
+use BuildEmpire\Apartment\Exceptions\SchemaCannotBePublicException;
+use BuildEmpire\Apartment\Exceptions\SchemaDoesntExistException;
+use BuildEmpire\Apartment\Exceptions\SchemaNameNotValidException;
+use BuildEmpire\Apartment\Schema;
+use Illuminate\Support\Facades\Schema as LumenSchema;
+use Carbon\Carbon;
+use BuildEmpire\Apartment\Helpers\ApartmentHelpers;
+
+class ArtisanApartmentCommands
+{
+    const ARTISAN = 1;
+    const WEB = 2;
+    const PUBLIC_SCHEMA = 'public';
+
+    protected $schema = false;
+
+    public function __construct(Schema $schema)
+    {
+        $this->schema = $schema;
+    }
+
+    public function makeSchema($schemaName) {
```

Comment:
It can't follow it, we have 2 functions here, `__construct` and `makeSchema`, one has the opening brace `{` on a new line, and one doesn't. 

Also in other places in other files there are spaces missing after parameters within a function call etc. I'm not sure your IDE is formatting for you.